js/ui/scroll_view/ui.scrollable.simulated.js

```diff
@@ -183,10 +185,34 @@ var Scroller = Class.inherit({
 
     _moveContent: function() {
         var location = this._location;
-        this._$container[this._scrollProp](-location);
+
+        this._$container[this._scrollProp](-location / this._getScaleRatio());
         this._moveContentByTranslator(location);
     },
 
+    _getScaleRatio: function() {
+        var element = this._$element.get(0),
+            ratio = windowUtils.hasWindow && math.round(element.getBoundingClientRect()[this._dimension]) / this._getComputedDimension(element, this._dimension) || 1;
```

Comment:
This line is long. Maybe, break it into a two line.

Reply:
... or extract some intermediate calculation(s) to separate variable(s).